packages/core/src/components/carousel/carousel.tsx

```diff
@@ -41,17 +42,25 @@ export class AtomCarousel {
   @Prop({ mutable: true })
   paginationClickable?: boolean
   @Prop({ mutable: true })
-  paginationType?: PaginationOptions['type'] = 'bullets'
+  paginationType?: PaginationOptions['type'] | 'thumbnails' = 'bullets'
```

Comment:
(question) this name `thumbnails` is your choice, right? 
In code below is verified and used `custom`, what do you think about using 'custom'

Reply:
the custom is because of the default value in the library, so, to not have a generic pagination type in our own carousel, if i use directly custom in the prop, we'll lost this type because it always render the thumbnails, so i believe that makes more sense create a different type that allows use both

Reply:
tks for explanation!